ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml

```diff
@@ -309,8 +307,6 @@ Item {
                             onClicked: {
                                 if (mouse.button === Qt.RightButton) {
```

Comment:
This has no purpose then :)

Reply:
It will very very soon! I'm adding options to copy tx hash and view on etherscan